charter/index.html

```diff
@@ -263,7 +263,9 @@ <h3>
         <section id="dependencies">
           <h3>Dependencies</h3>
 
-          <p>Depending on the Working Group progress, including consideration for <a href="https://www.w3.org/2021/Process-20211102/#adequate-implementation">adequate implementation experience</a>, the Group will also produce W3C Recommendations for the following documents:</p>
+          <p>The proposed adopted draft for the Solid Protocol v1.0 has a number of normative dependencies with a maturity level insufficient for a recommendation (Community Group reports, Editor's Drafts). The Working Group will need to decide how to deal with these dependencies in order to allow the Solid Protocol v1.0 to progress on the Recommendation track. This may include downgrading those dependencies to non-normative references, specifiying the required parts in the Solid Protocol itself, of relying on other specifications to fulfil the same purpose.</p>
```

Comment:
v1.0 doesn't exist and I think we should revise the reference to this altogether.

```suggestion
          <p>The proposed adopted draft for the Solid Protocol has a number of normative dependencies with a maturity level insufficient for a recommendation (Community Group reports, Editor's Drafts). The Working Group will need to decide how to deal with these dependencies in order to allow the Solid Protocol v1.0 to progress on the Recommendation track. This may include downgrading those dependencies to non-normative references, specifying the required parts in the Solid Protocol itself, of relying on other specifications to fulfil the same purpose.</p>
```